Libplanet.Explorer/GraphTypes/TransactionType.cs

```diff
@@ -33,12 +33,12 @@ public TransactionType()
                 name: "BlockRef",
                 resolve: ctx =>
                 {
-                    var blockChainContext = (IBlockChainContext<T>)ctx.UserContext;
-                    if (blockChainContext.Store is RichStore richStore)
+                    // FIXME: use store with DI.
+                    if (ctx.UserContext[nameof(IBlockChainContext<T>.Store)] is RichStore richStore)
```

Comment:
Is it safe when `ctx.UserContext` don't have `nameof(IBlockChainContext<T>.Store)` as key? or, we don't need to mind that case?

Reply:
It is not sure that it has `nameof(IBlockChainContext<T>.Store)` as a key or not, in compile time.
So it needs runtime test manually and I checked it with planetarium/libplanet-explorer-frontend successfully.

If we tried to look for something to make us trust the code, I may say we execute all queries manually in the endpoint and we are passing the values at the below codes.

```csharp
...
        [HttpPost("/graphql/")]
        public async Task<IActionResult> GetGraphQLResult(
            [FromBody] GraphQLBody body
        )
        {
            var json = await _schema.ExecuteAsync(_ =>
            {
                _.UserContext = new Dictionary<string, object>
                {
                    [nameof(_context.Store)] = _context.Store,
                    [nameof(_context.BlockChain)] = _context.BlockChain,
                };
                _.Query = body.Query;
                _.ThrowOnUnhandledException = true;
                if (body.Variables != null)
                {
                    _.Inputs = body.Variables.ToString(Newtonsoft.Json.Formatting.None).ToInputs();
                }
            });
            return Ok(JObject.Parse(json));
        }
```